mitmproxy/tools/console/overlay.py

```diff
@@ -60,14 +65,15 @@ def keypress(self, size, key):
 
 
 class ChooserListWalker(urwid.ListWalker):
-    def __init__(self, choices, current):
+    def __init__(self, choices, current, shortcuts):
         self.index = 0
         self.choices = choices
         self.current = current
+        self.shortcuts = shortcuts
```

Comment:
Let's use a class variable as in https://github.com/mitmproxy/mitmproxy/pull/2834#pullrequestreview-93365506 - no need to pass and slice the same string all the time.